fuzzers/structure_aware/nautilus_sync/src/bin/libafl_cc.rs

```diff
@@ -2,7 +2,6 @@ use std::env;
 
 use libafl_cc::{ClangWrapper, CompilerWrapper, ToolWrapper};
 
-#[allow(clippy::missing_panics_doc)]
```

Comment:
pretty sure this is needed you build this with `clippy -- -D clippy::pedantic` (which I like to do)
Anyway it doesn't hurt right?